### DISCUSSION TITLE
Translation Status: German

### DISCUSSION THREAD

**Discussion by korvegan** [Lurkars/gloomhavensecretariat#487] [Q&A]
Hi @Lurkars i thought, i will create a new topic regarding the german translation overall.

I created a pull-request for the new 

`data/fh/label/de.json` File

label-spoiler will be next and what about the crossover character sheets?

**Comment by Lurkars** [comment]
Will add crossover later today.

**Reply by korvegan** [reply]
Is it correct that the crossover have no ability cards? 
![Screenshot_20240106_132036_Chrome](https://github.com/Lurkars/gloomhavensecretariat/assets/8201257/15c8f712-6644-424b-a91f-649981f563ad)

**Reply by Lurkars** [reply]
This seems a bug I introduced with an update. Only Brute and Mindthief (GH edition) have deck now and this is not a feature suported by now. I will fix this!
